### PR TITLE
[754975] [Global Search Results] Added description to VoiceOver for search results

### PR DESCRIPTION
**Pull request by Semptra**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/754975

Things that are done in this PR:
* Added `Description` to `AccessibilityMessage` for `SearchResult` class
* Added `Description` to `AccessibilityMessage` in overrides for `DeclaredSymbolInfoResult` and `FileSearchResult`

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs [hunk 1]
```diff
@@ -47,44 +47,26 @@ class DeclaredSymbolInfoResult : SearchResult
...
+						NavigateToItemKind.Enum		 => "Enumeration",
+						NavigateToItemKind.Constant  => "Constant",
+						NavigateToItemKind.Field	 => "Field",
+						NavigateToItemKind.EnumItem  => "Enumeration",
+						NavigateToItemKind.Interface => "Interface",
+						NavigateToItemKind.Method	 => "Method",
+						NavigateToItemKind.Property  => "Property",
+						NavigateToItemKind.Structure => "Structure",
+						_							 => null
+					};
```
Comment (rodrmoya) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs]: Can you fix formatting here?

Comment (Semptra) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs]: @rodrmoya what should I use for formatting for the `switch` expression? Here I used tabs to align all `=>` operators on the same level. Should I use spaces for alignment of just leave everything with one space?

Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs]: I would just use `case => result`, that is a single space, but whatever, they should use the same, here on GH it looks badly formatted. If that's not the case on the editor, ignore me

Comment (Semptra) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs]: Fixed

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs [hunk 2]
```diff
@@ -47,44 +47,26 @@ class DeclaredSymbolInfoResult : SearchResult
...
-					case NavigateToItemKind.Property:
-						return GettextCatalog.GetString ("Property {0}", result.Name);
-
-					case NavigateToItemKind.Structure:
-						return GettextCatalog.GetString ("Structure {0}", result.Name);
-					default:
-						return result.Name;
-					}
+					string resultKind = result.Kind switch  {
+						NavigateToItemKind.Class     => "Class",
```
Comment (sevoku) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.MainToolbar/RoslynSearchCategory.DeclaredSymbolInfoResult.cs]: This won't be picked up by the translation parser

**Pull request closed without merging**